api/src/pcapi/routes/serialization/venues_serialize.py

```diff
@@ -32,6 +32,24 @@
 MAX_LATITUDE = 90
 
 
+class DMSApplicationForEAC(BaseModel):
+    venueId: int
+    state: str
+    procedure: int
+    application: int
+    lastChangeDate: datetime
+    depositDate: datetime
+    expirationDate: datetime
```

Comment:
ce champ est nullable (je l'ai appris a mes dépends ce matin :sob: )
```suggestion
    expirationDate: datetime | None
```